cmd/cluster/kubevirt/params/params.go

```diff
@@ -0,0 +1,202 @@
+package params
```

Comment:
this should be unit tested, shouldn't it ?

Reply:
Have copy it verbatim from kubevirt/kubevirt, there is no unit test there, don't know if this justify not adding unit test.

Since we are already unit testing create cluster that can be enough.

Reply:
> Since we are already unit testing create cluster that can be enough.

well, I don't think that's how unit tests are supposed to work. Have you checked this file's coverage ? 

Not saying it's a must, but it's good practice.